.teamcity/settings.kts

```diff
@@ -1,37 +1,46 @@
 import jetbrains.buildServer.configs.kotlin.v2019_2.*
-import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nuGetPublish
 import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.ScheduleTrigger
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.schedule
 import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
 
-version = "2022.04"
-
 project {
-    buildType(RhinoLicensingBuild)
+    buildType(RhinoLicensing)
+    buildType(RhinoLicensingSchd)
+    buildType(RhinoLicensingQA)
 }
 
-object RhinoLicensingBuild : BuildType({
-    name = "Rhino.Licensing Build"
+object RhinoLicensing : BuildType({
+    id("RhinoLicensing")
```

Comment:
Just to be clear, changing this will lose the history under the old ID. Are we ok with that?